core/src/main/java/com/google/zxing/maxicode/decoder/DecodedBitStreamParser.java

```diff
@@ -144,7 +144,7 @@ private static String getPostCode3(byte[] bytes) {
          SETS[0].charAt(getInt(bytes, new byte[] {27, 28, 29, 30, 19, 20})),
          SETS[0].charAt(getInt(bytes, new byte[] {21, 22, 23, 24, 13, 14})),
          SETS[0].charAt(getInt(bytes, new byte[] {15, 16, 17, 18,  7,  8})),
-         SETS[0].charAt(getInt(bytes, new byte[] { 9, 10, 11, 12,  1,  2})),
+         SETS[0].charAt(getInt(bytes, new byte[] { 9, 10, 11, 12,  1,  2}))
```

Comment:
It appears trailing commas are now allowed in JavaScript, not sure what I was running into, too many late nights, may have been seeing things. My arguments still stand, but I've gone ahead and reverted the trailing comma edits so now the PR is just whitespace.